src/index.html

```diff
@@ -13,6 +13,38 @@
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day-sun calendar--month-length-31">
```

Comment:
Using a lot of `div` elements with the same class manually to represent days is not maintainable or scalable. Consider using a templating engine or JavaScript to generate these elements dynamically based on the month length and start day.